tools/hostbasedimagenames/main.go

```diff
@@ -0,0 +1,40 @@
+package hostbasedimagenames
```

Comment:
Could this name of `main.go` be changed? I feel it is improper to name a package file as `main.go`

Reply:
> Could this name of `main.go` be changed?

I've renamed it to `hostbasedimagenames.go` with e3e4231 → 3bb506e.